src/features/toast/ToastRenderer.tsx

```diff
@@ -1,41 +1,49 @@
-import React, { useRef } from 'react'
-import { useSelector } from 'react-redux'
+import React, { useEffect, useRef } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
 import { ToastContainer, toast } from 'react-toastify'
 
 import { Run, RunStatus } from '../../qri/run'
 import Toast from './Toast'
 import { selectRunsFromEventLog } from '../workflow/state/workflowState'
-
+import { loadVersionInfo } from "../collection/state/collectionActions";
 
 import 'react-toastify/dist/ReactToastify.css'
 
 const ToastRenderer: React.FC<{}> = () => {
 
   const runs = useSelector(selectRunsFromEventLog)
-
+  const dispatch = useDispatch()
   // use a ref to store the workflow status so we can
   // determine when it changes and update an already visible toast
   const runsRef = useRef({} as Record<string, RunStatus>)
+  const initIDs = runs.map((r: Run) => r.initID)
 
   // artifically delay the update to 'success'
   const SUCCESS_START_DELAY = 0
 
   // after 'success', close the toast 4 seconds later
   const SUCCESS_CLOSE_DELAY = 4000
 
+  useEffect(() => {
+    initIDs.forEach((initId) => {
```

Comment:
we should be adding type to as much as we can:

`initIDs.forEach((initID: string) => {`